### PR TITLE
add config updates for current sprint env

### PR DESCRIPTION
**Pull request by ladamesny**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

File: scripts/deploy-bitte [hunk 1]
```diff
@@ -26,6 +26,7 @@ fi
 declare -A ref_env_mapping=(
     [production]=production
     [main]=staging
+    [sprint-*]=currentSprintMarlowe
```
Comment (shlevy) [inline scripts/deploy-bitte:29]: See https://github.com/input-output-hk/marlowe-cardano/pull/84/files#diff-c69d8b18d9f3cd934d935bf02255f1d1746031d582d2ef8d4cf799e927ddee1aL34 , we currently just do a direct hash lookup which will fail here. We can probably do something similar to https://github.com/input-output-hk/plutus-apps/blob/main/scripts/deploy-bitte#L19-L21 to transform `ref` prior to doing the lookup

**Review by shlevy — CHANGES_REQUESTED** [review]
See above comment

File: scripts/deploy-bitte (hunk 1 above)
Comment (ladamesny) [reply scripts/deploy-bitte:30]: Thanks @shlevy 

I copied over that ref regex from the bitte file in plutus apps, however I'm not sure if understanding this file correctly.

Is `ref` supposed to reference a branch name from the marlowe-cardano repo? Then then we use it as a key to find a matching environment in the ref_env_mapping?

Comment (shlevy) [reply scripts/deploy-bitte:30]: The idea is that we look up `ref` in `ref_env_mapping`. So we don't want to copy the regex exactly from plutus-apps, there they want to match on a tag of the form `vYYYY-MM-DD` so the regex transforms any tag matching that format into the literal string "vYYYY-MM-DD", which string is then stored in ref_env_mapping.

So you'd want a regex that replaces sprint-12 with "sprint-N" or something and then store "sprint-N" in the mapping.

Comment (ladamesny) [reply scripts/deploy-bitte]: @shlevy ok thank you - that's clearer for me. Ok so I updated the regex. I'm basically replacing `sprint-12` to `sprint-` - it was just an easier regex to write. 

So I updated the command - I feel good about `| sed -e 's/[0-9]//g` part as I tested it on the terminal, but I'm not sure about the portion before it. What's the best way to test that `ref="${ref}sprint-${[0-9]}"` would match `sprint-12`?

File: scripts/deploy-bitte [hunk 2]
```diff
@@ -16,7 +16,8 @@ then
     error "GITHUB_SHA environment variable not set!"
 fi
 
-readonly ref="${GITHUB_REF_SLUG:-}"
+ref="${GITHUB_REF_SLUG:-}"
+ref="${ref}sprint-${[0-9]} | sed -e 's/[0-9]//g'"
```
Comment (shlevy) [inline scripts/deploy-bitte:20]: This isn't right. You can test this out locally by doing:

```
$ ref=sprint-23
$ ref=<expression you want to test>
$ echo $ref
```

I recommend just using the bash replacement style like I did in plutus-apps. See https://tldp.org/LDP/abs/html/string-manipulation.html#SUBSTRREPL00

**Review by shlevy — CHANGES_REQUESTED** [review]

File: scripts/deploy-bitte (hunk 2 above)
Comment (ladamesny) [reply scripts/deploy-bitte]: ok I think I got it now - the new regex works locally. Thanks for your help on this @shlevy

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -7,6 +7,7 @@ on:
       - deploy-hernan
       - deploy-pablo
       - shlevy
+      - sprint-*
```
Comment (ladamesny) [inline .github/workflows/deploy.yml:10]: oops, I guess I don't need this star anymore - I'm going to remove

Comment (ladamesny) [reply .github/workflows/deploy.yml:10]: or should I @shlevy ? I think I need this. Going to leave this alone till I hear from you.

**Review by shlevy — APPROVED** [review]

**Pull request merged**